velox/docs/develop/testing/fuzzer.rst

```diff
@@ -164,21 +164,27 @@ input vector and expression to files and replay these later.
 
 2. Run Expression Runner using generated files.
 
-``--repro_persist_path <path/to/directory>`` flag tells the Fuzzer to save
-input vector and expression SQL to files in the specified directory and print
-out the exact paths. Fuzzer uses :doc:`VectorSaver <../debugging/vector-saver>` for storing vectors on disk
-while preserving encodings.
-
-ExpressionRunner takes a path to input vector, path to expression SQL and
-"mode" and evaluates the specified expression on the specified data.
+``--repro_persist_path <path/to/directory>`` flag tells the Fuzzer to save the
+input vector, initial result vector, expression SQL, and other relevant data to files in a new directory saved within
+the specified directory. It also prints out the exact paths for these. Fuzzer uses :doc:`VectorSaver <../debugging/vector-saver>`
+for storing vectors on disk while preserving encodings.
+
+ExpressionRunner needs at the very least a path to input vector and path to expression SQL to run.
+However, you might need more files to reproduce the issue. All of which will be present in the directory
+that the fuzzer test generated. You can directly point the ExpressionRunner to that directory using --input_dir
+where it will pick up all the files automatically or you can specify each explicitly using other startup flags.
 ExpressionRunner supports the following flags:
 
+* ``--input_dir`` directory path where all input files (required to reproduce a failure) that are generated by the Fuzzer are expected to reside. ExpressionRunner will automatically pick up all the files from this folder unless they are explicitly specified via their respective startup flag.
```

Comment:
input_path and input_dir are too similar; is there a way to distinguish these a bit more?

--fuzzer_repro_path?